TwilioVerifyDemo/TwilioVerifyDemo.xcodeproj/project.pbxproj

```diff
@@ -8,6 +8,9 @@
 
 /* Begin PBXBuildFile section */
 		860BB68C262A018A00E021C9 /* TwilioVerifySDK in Frameworks */ = {isa = PBXBuildFile; productRef = 860BB68B262A018A00E021C9 /* TwilioVerifySDK */; };
+		97B7832A26F0FA9E0066CFD1 /* (null) in Sources */ = {isa = PBXBuildFile; };
+		97B7832C26F1005F0066CFD1 /* PushChallengePayloadData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 97B7832B26F1005F0066CFD1 /* PushChallengePayloadData.swift */; };
+		97C5984426EA5AB0006C6562 /* (null) in Sources */ = {isa = PBXBuildFile; };
```

Comment:
We should remove those null references.

Reply:
My bad, deleting the NotificationExtension leave this null references, I'll delete them.